change/@fluentui-web-components-15df2812-4bc2-40b6-a669-d55fe7fff449.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31694&commitId=5d08b813c009a8de36b496ee02945b54b9867fba&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|

</details>


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.appearance - RTL.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled - RTL.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - RTL.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - RTL.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled - RTL.selected.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>